product-mini/platforms/posix/main.c

```diff
@@ -570,6 +570,22 @@ main(int argc, char *argv[])
         goto fail3;
     }
 
+#if WASM_ENABLE_DEBUG_INTERP != 0
+    if (ip_addr != NULL) {
+        wasm_exec_env_t exec_env =
+            wasm_runtime_get_exec_env_singleton(wasm_module_inst);
+        if (exec_env == NULL) {
+            printf("%s\n", wasm_runtime_get_exception(wasm_module_inst));
+            goto fail4;
+        }
+        uint32_t debug_port = wasm_runtime_start_debug_instance(exec_env);
```

Comment:
Had better declare variables at the beginning of a scope, otherwise some compiler with `C90` may complain

Reply:
fixed. thank you.